cmd/mindthegap/create/bundle/bundle.go

```diff
@@ -38,12 +39,13 @@ import (
 
 func NewCommand(out output.Output) *cobra.Command {
 	var (
-		imagesConfigFile     string
-		helmChartsConfigFile string
-		platforms            = flags.NewPlatformsValue("linux/amd64")
-		outputFile           string
-		overwrite            bool
-		imagePullConcurrency int
+		imagesConfigFile       string
+		helmChartsConfigFile   string
+		ociArtifactsConfigFile string
```

Comment:
I am wondering if we could use a single file with `oci://` prefix.

```
docker.io/library/alpine:v1
oci://ghcr.io/mesosphere/charts/kommander:v2
```

Reply:
iiuc, currently the imagesConfigFile can already take oci images. However, it is OCI Artifacts, which usually don't contain platform information, that is breaking the bundling process. 

talked to Jimmi yesterday and we decided to get something work first without changing the logic of indexing container images.